services/mailer/mail.go

```diff
@@ -171,3 +171,39 @@ func fromDisplayName(u *user_model.User) string {
 	}
 	return u.GetCompleteName()
 }
+
+func generateMetadataHeaders(repo *repo_model.Repository) map[string]string {
+	return map[string]string{
+		// https://datatracker.ietf.org/doc/html/rfc2919
+		"List-ID": fmt.Sprintf("%s <%s.%s.%s>", repo.FullName(), repo.Name, repo.OwnerName, setting.Domain),
```

Comment:
I can see similar code from other part.

But I guess we could try to merge the duplicate code, and the usage of `FullName` seems fragile. What if the "FullName" contains `>`

Reply:
Oops, the code has been de-duplicated. Sorry for the misreading.